web_src/css/modules/table.css

```diff
@@ -167,6 +167,12 @@
   text-overflow: ellipsis;
 }
 
+.ui.selectable.table > tbody > tr:hover,
+.ui.table tbody tr td.selectable:hover {
+  background: var(--color-hover);
+  color: var(--color-text);
```

Comment:
Maybe we should remove this `color`, in case the TD already has some colors like "red"